src/main/java/com/exasol/adapter/installer/UserInput.java

```diff
@@ -0,0 +1,19 @@
+package com.exasol.adapter.installer;
+
+import java.util.Map;
+
+import lombok.Getter;
+
+@Getter
```

Comment:
I just added it for development time, because we're adding and renaming fields too often. I can add a normal builder and getter when the structures are stable